### PR TITLE
Fix deploy with dbinit

### PR DESCRIPTION
**Pull request by calvarez-ov**
Add the db migration script steps to supervisord.conf, do they don't have to be executed manually.

### CODE REVIEW

File: deploy/commons.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Django>=1.9,<1.9.99
+Django>=1.8,<1.8.99
```
Comment (calvarez-ov) [inline deploy/commons.txt]: The `python3 manage.py makemigrations` and `python3 manage.py migrate` scripts didn't actually end up creating the `gm_pr_*` tables with Django 1.9.

File: deploy/Dockerfile [hunk 2]
```diff
@@ -21,7 +21,8 @@ RUN apt-get install -y \
     supervisor \
     apache2 \
     libapache2-mod-wsgi-py3 \
-    python3-pip
+    python3-pip \
+    sqlite3
```
Comment (calvarez-ov) [inline deploy/Dockerfile]: `sqlite3` is needed even if we don't invoke it directly.  It is needed by `python3 manage.py dbshell` in supervisord.conf

File: deploy/supervisord.conf [hunk 3]
```diff
@@ -39,3 +39,7 @@ priority=50
 command=python3 manage.py collectstatic --noinput
 user=www-data
 priority=60
+
+[program:migrate]
+command=/bin/bash -c "sleep 15; python3 manage.py makemigrations && python3 manage.py migrate && cat gm_pr/projects.sql |python3 manage.py dbshell"
+priority=70
```
Comment (calvarez-ov) [inline deploy/supervisord.conf]: Info: I initially had a line `user=www-data`, but that resulted in permission errors.

File: deploy/Dockerfile (hunk 2 above)
Comment (caarmen) [inline deploy/Dockerfile]: Irrelevant now. This was reverted.

**Comment by CedricCabessa** [comment]
replaced by the big one

**Pull request closed without merging**